libibverbs/man/ibv_advise_mr.3.md

```diff
@@ -73,7 +73,7 @@ application performance.
 **ibv_advise_mr()** returns 0 when the call was successful, or the value
 		    of errno on failure (which indicates the failure reason).
 
-*ENOSYS*
+*EOPNOTSUPP*
```

Comment:
The man pages should specify that ENOSYS may sometimes be returned by old versions of the library.

Reply:
Thanks for pointing that out, I'll post a fix soon.